uber/site_sections/registration.py

```diff
@@ -1,18 +1,10 @@
 from uber.common import *
 
-def check_everything(attendee):
```

Comment:
Oh wow, did you go through and pull out those bits? Thank you! I hated looking at them. :P